data/json/items/newspaper.json

```
@@ -45,6 +45,22 @@
     "weight": "12 mg",
     "volume": "1 ml"
   },
+  {
+    "type": "GENERIC",
+    "id": "childnote",
+    "category": "books",
+    "symbol": ",",
+    "color": "white",
+    "name": { "str": "child's note" },
+    "snippet_category": "note_child",
+    "description": "A piece of paper.  Something's written on it, scrawled in child-like handwriting.",
```

```suggestion
    "name": { "str": "child's note" },
    "snippet_category": "note_child",
    "description": "A piece of paper.  Something's written on it, scrawled in simple, messy handwriting.",
```
If a [X] wrote something, it's not [X]-*like* 🤔

---

Do you want to hear a secret? I knew someone would correct this, I just had no better idea on how to describe the handwriting lol. Thanks for this, though. My grammar is relatively limited.